include/extractor/edge_based_graph_factory.hpp

```diff
@@ -70,8 +72,12 @@ class EdgeBasedGraphFactory
 
     unsigned GetHighestEdgeID();
 
-    TurnInstruction
-    AnalyzeTurn(const NodeID u, const NodeID v, const NodeID w, const double angle) const;
+    TurnInstruction AnalyzeTurn(const NodeID u,
+                                const EdgeID e1,
+                                const NodeID v,
+                                const EdgeID e2,
+                                const NodeID w,
+                                const double angle) const;
```

Comment:
This is

       e1     e2
    u ---- v ---- w

I assume? So this only takes the immediate neighbours into account?